modules/i18n/consts.js

```diff
@@ -0,0 +1 @@
+export const customNamespace = 'knossos-i18n'
```

Comment:
... is there really need to separate this into own file? I don't think there is.